internal/db/schema/migrations/oss/postgres/79/01_auth_oidc_prompt.up.sql

```diff
@@ -0,0 +1,105 @@
+-- Copyright (c) HashiCorp, Inc.
+-- SPDX-License-Identifier: BUSL-1.1
+
+begin;
+
+-- auth_oidc_prompt_enm entries define the supported oidc auth method prompts.
+create table auth_oidc_prompt_enm (
+  name text primary key
+    constraint only_predefined_auth_oidc_prompts_allowed
+    check (
+      name in (
```

Comment:
The constraint needs to support validation for  comma separated values. Maybe we could use regex for this

Reply:
Why does it need to support a comma delimited list.  Isn't there a one to many
relationship between the auth method and it's associated prompts?